src/encoded/static/components/matrix_human_donor.js

```diff
@@ -233,13 +237,18 @@ const convertToMatrixData = (dataTableData, baseUrl, searchBaseUrl, graph) => {
                     const itemKey = `${accession}-${assayCountIndex}`;
                     const assayTitle = x.header[assayCountIndex];
 
+                    const searchBaseUrlQuery = new QueryString(searchBaseUrl);
```

Comment:
Really supposed to separate out the query string from the rest of the URL and pass that to `new QueryString`, but in most (not all) cases it works without doing that. I feel like I need to add some code to the QueryString class to optionally accept a whole URI and split the query string off itself, because people keep passing whole URLs to it.

Reply:
Another good point.  There could be multiple assay_titles, so, it would require fancy javascript to get them all out given they could be in any location. The approach I took is a little janky but simpler. Is it acceptable?